dpnp/tests/test_sycl_queue.py

```diff
@@ -119,42 +74,28 @@ def vvsort(val, vec, size, xp):
     ],
 )
 @pytest.mark.parametrize(
-    "device",
-    valid_devices,
-    ids=[device.filter_string for device in valid_devices],
+    "device", valid_dev, ids=[dev.filter_string for dev in valid_dev]
 )
 def test_array_creation(func, arg, kwargs, device):
     dpnp_kwargs = dict(kwargs)
     dpnp_kwargs["device"] = device
-    dpnp_array = getattr(dpnp, func)(*arg, **dpnp_kwargs)
-
-    numpy_kwargs = dict(kwargs)
-    numpy_kwargs["dtype"] = dpnp_array.dtype
-    numpy_array = getattr(numpy, func)(*arg, **numpy_kwargs)
-
-    assert_dtype_allclose(dpnp_array, numpy_array)
-    assert dpnp_array.sycl_device == device
+    x = getattr(dpnp, func)(*arg, **dpnp_kwargs)
+    assert x.sycl_device == device
 
 
 @pytest.mark.parametrize(
-    "device",
-    valid_devices,
-    ids=[device.filter_string for device in valid_devices],
+    "device", valid_dev, ids=[dev.filter_string for dev in valid_dev]
 )
 def test_empty(device):
-    dpnp_array = dpnp.empty((2, 2), device=device)
-    assert dpnp_array.sycl_device == device
+    x = dpnp.empty((2, 2), device=device)
+    assert x.sycl_device == device
 
 
 @pytest.mark.parametrize(
-    "device_x",
-    valid_devices,
-    ids=[device.filter_string for device in valid_devices],
+    "device_x", valid_dev, ids=[dev.filter_string for dev in valid_dev]
 )
 @pytest.mark.parametrize(
-    "device_y",
-    valid_devices,
-    ids=[device.filter_string for device in valid_devices],
+    "device_y", valid_dev, ids=[dev.filter_string for dev in valid_dev]
 )
 def test_empty_like(device_x, device_y):
```

Comment:
Can we move it to `test_array_creation_follow_device`, because we no need to compare the result array?